src/styles/colors.js

```diff
@@ -1,21 +1,31 @@
 export default {
-    white: '#FFFFFF',
-    gray1: '#FAFAFA',
-    gray2: '#ECECEC',
-    gray3: '#C6C9CA',
-    gray4: '#7D8B8F',
     dark: '#0b1b34',
     black: '#000000',
-    blue: '#0185ff',
-    blueHover: '#0063bf',
+    blue: '#5AB0FF',
+    blueHover: '#B0D9FF',
     floralwhite: '#fffaf0',
-    green: '#03d47c',
-    greenHover: '#03c775',
-    orange: '#ff7101',
-    pink: '#f68dfe',
-    red: '#fc3826',
-    redHover: '#e13826',
-    redDisabled: '#fea29a',
-    yellow: '#fed607',
+    orange: '#FF7101',
+    pink: '#F68DFE',
+    green: '#03D47C',
+    greenHover: '#00C271',
+    greenPressed: '#35DD96',
+    greenDisabled: '80E9BD',
+    red: '#F25730',
+    redHover: '#DE4822',
+    redPressed: '#F57959',
+    redDisabled: '#F8AA97',
+    yellow: '#FED607',
     transparent: 'transparent',
+
+    // Dark Mode Theme Colors
+    greenAppBackground: '#061B09',
+    greenHighlightBackground: '#002E22',
+    greenBorders: '#1B5744',
+    greenIcons: '#8B9C8F',
+    greenSupportingText: '#C9D3C5',
+    white: '#FFFFFF',
+    greenDefaultButton: '#184E3D',
+    greenDefaultButtonHover: '#2C6755',
+    greenDefaultButtonPressed: '#467164',
+    greenDefaultButtonDisabled: '#8BA69E',
```

Comment:
I think we could just use the color scale we have in Figma for our brand colors:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/2319350/200708973-856b3910-54c8-4227-9dd1-f0c0f2ce17a4.png">


But perhaps we need to get a little creative with some of the custom values we're using for button hover and button pressed and try to change some to match our color scale. For example, maybe our default button color can just use our green700 color. And then we can add a new color called green650 for buttonHover. 

<img width="983" alt="image" src="https://user-images.githubusercontent.com/2319350/200709635-fed3f336-0b32-4c19-856e-5c16935c9352.png">


For button pressed, I was thinking we would just make them 80% opacity as I think react kind of already this does for us? And then disabled buttons just get 50% opacity so no need to add a color there either.

Reply:
I think right now most of the dark mode theme colors aren't a part of that scale. Right now, the only two that overlap are success/green400 and greenHighlightBG/green800. I'm down for custom color names, but it might be worth renaming them in Figma first?

Reply:
I don't think I am following you - mind explaining what you mean by custom color names and renaming them in Figma? 

I'd also be down to just keep these values as you had them (where they are slightly semantic) - I think that's okay and that's actually how we have them (the ones not right on our green scale) named in Figma anyways.

Reply:
My only concern with leaving these the way they are now is that we may end up using a color of green (called greenDefaultButton for instance) on an element that's not a button. I'd almost prefer these colors to have totally arbitrary names, like `leafGreen` in the color file, and only have them pick up semantic meaning in the `theme.js` file.

With that said, this isn't a blocker for me. If you don't agree with my reasoning, then we can move forward as is.

Reply:
I would prefer not to go down a rabbit hole for naming things like leafGreen - the numeric scale makes way more sense as it takes away the subjectivity of the descriptor words.

Reply:
The overall structure would be 
| colors.js | theme.js | styles.js |
|:---:|:---:|:---:|
| where colors are defined | where colors pick up semantic meaning | where semantic colors are used in styles |

If we ever need to swap a whole color, we go to colors.js. If we ever want to change just one color in the theme but don't want to change it across the board, we go to themes.js (e.g. two separate elements have an orange color and we want to swap one, but leave it for the other one), lastly, if we want to change the style for a single component, we go to styles.js.

Reply:
OK we just talked on a call about this one and the resolution is that we should probably just remove the colors.js file altogether and have the hex codes live in the theme.js file. This feels like polish though, so we're gonna wait to make a different PR about that later when we're installing the linting rules and finalizing the project.